packages/react-table/src/components/TableComposable/Th.tsx

```diff
@@ -24,6 +25,8 @@ export interface ThProps
   dataLabel?: string;
   /** Renders a checkbox select so that all row checkboxes can be selected/deselected */
   select?: ThSelectType;
+  /** Renders a chevron so that all row chevrons can be expanded/collapsed */
+  collapse?: ThCollapseType;
```

Comment:
I think `ThCollapseType` is confusing since for Td we use `TdExpandType`. Could those be consistent?

Reply:
sure